gallery/src/GalleryNav/GalleryNav.tsx

```diff
@@ -21,43 +21,75 @@ import {
   complement,
   values
 } from 'ramda';
-import { NxTreeView, NxTreeViewChild, NxFilterInput, NxTreeViewProps } from '@sonatype/react-shared-components';
+import {
+  NxCollapsibleItems,
+  NxFilterInput,
+  NxCollapsibleItemsProps,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
 
-import pageConfig from './pageConfig';
+import pageConfig from '../pageConfig';
 import { useLocation } from 'react-router';
 import { NavLink } from 'react-router-dom';
-import { PageMapping, PageConfig } from './pageConfigTypes';
-import { markByFilter, matchesFilter } from './filterUtil';
-
-const renderLinks = (filter?: string) => pipe(
-    keys,
-    map((pageName: string) =>
-      <NxTreeViewChild key={pageName}>
-        <NavLink to={`/pages/${pageName}`} activeClassName="selected">
-          {markByFilter(filter, pageName)}
-        </NavLink>
-      </NxTreeViewChild>
-    )
-);
+import { PageMapping, PageConfig, PageType } from '../pageConfigTypes';
+import { markByFilter, matchesFilter } from '../filterUtil';
+
+import { faFile, faCode, faRulerCombined, IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import { faReact, faSass, faJs } from '@fortawesome/free-brands-svg-icons';
+
+import './GalleryNav.scss';
+
+const PAGE_TYPE_TO_ICON_MAP: { [page in PageType]: IconDefinition } = {
```

Comment:
I'm not getting type errors without this annotation
```suggestion
const PAGE_TYPE_TO_ICON_MAP = {
```